packages/lib/server-only/document/search-documents-with-keyword.ts

```diff
@@ -0,0 +1,66 @@
+import { prisma } from '@documenso/prisma';
+
+export type SearchDocumentsWithKeywordOptions = {
+  query: string;
+  userId: number;
+};
+
+export const searchDocumentsWithKeyword = async ({
+  query,
+  userId,
+}: SearchDocumentsWithKeywordOptions) => {
+  const user = await prisma.user.findFirstOrThrow({
+    where: {
+      id: userId,
+    },
+  });
+
+  const documents = await prisma.document.findMany({
```

Comment:
Looks like we're missing a limit here, if I were to search for `a` I might return thousands of results

Reply:
Ideally this would be an option passed into the method

Reply:
done now this takes a `limit` variable with a default value of 5.